src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -576,5 +688,25 @@ public static uint RotateRight(uint value, int offset)
         [CLSCompliant(false)]
         public static ulong RotateRight(ulong value, int offset)
             => (value >> offset) | (value << (64 - offset));
+
+        /// <summary>
+        /// Rotates the specified value right by the specified number of bits.
+        /// Similar in behavior to the x86 instruction ROR.
+        /// </summary>
+        /// <param name="value">The value to rotate.</param>
+        /// <param name="offset">The number of bits to rotate by.
+        /// Any value outside the range [0..31] is treated as congruent mod 32 on a 32-bit process,
```

Comment:
Would you improve the wording of the documentation comment (I am not a native English speaker)?